src/main/java/ch/njol/skript/SkriptAddon.java

```diff
@@ -23,46 +23,60 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
+import org.bukkit.Bukkit;
+import org.bukkit.plugin.Plugin;
 import org.bukkit.plugin.java.JavaPlugin;
 import org.eclipse.jdt.annotation.Nullable;
 
 import ch.njol.skript.localization.Language;
 import ch.njol.skript.util.Utils;
 import ch.njol.skript.util.Version;
+import org.jetbrains.annotations.ApiStatus;
+import org.jetbrains.annotations.NotNull;
+import org.skriptlang.skript.localization.Localizer;
+import org.skriptlang.skript.registration.SyntaxRegistry;
 
 /**
  * Utility class for Skript addons. Use {@link Skript#registerAddon(JavaPlugin)} to create a SkriptAddon instance for your plugin.
+ * @deprecated Use {@link org.skriptlang.skript.addon.SkriptAddon}.
  */
-public final class SkriptAddon {
+@Deprecated
+@ApiStatus.NonExtendable
```

Comment:
You don't need to bother with this annotation if all of the constructors are package-private, nothing could extend it anyway.